src/app/health/health.scss

```diff
@@ -21,5 +21,5 @@ mat-table {
   width: max-content;
 }
 .table-column {
-  min-width: $exam-table-col-width;
+  min-width: 100px;
```

Comment:
Instead of changing the css property, you should change the scss variable at the top of the file.  That variable is used in a couple of places to set the width of the columns.